### PR TITLE
1.18 - component pages

### PR DESCRIPTION
**Pull request by evilnick**

### CODE REVIEW

**Comment by evilnick** [comment]
components page preview - https://deploy-preview-387--cdk-docs-next.netlify.com/kubernetes/docs/1.18/components

**Comment by kwmonroe** [comment]
@evilnick the container images for 1.18 are listed here:

https://docs.google.com/document/d/1msmjMUa42xLUZMHFOys3YkdWBipauzoC-xBFwmeEsDA/edit#heading=h.17mut1c8mhsd

May have to do some tweaks, but the bulk of them are correct for 1.18.

File: pages/k8s/1.18/components.md [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+keywords: component, charms, versions, release
+tags:
+    - reference
+sidebar: k8smain-sidebar
+permalink: 1.18/components.html
+layout:
+    - base
+    - ubuntu-com
+toc: false
+bundle_revision: '426'
```
Comment (battlemidget) [inline pages/k8s/1.18/components.md:17]: This was recently updated b/c of the hacluster interface updated, I think you want this to be 428

**Comment by evilnick** [comment]
@tvansteenburgh this one is ready too

**Comment by evilnick** [comment]
:cowboy_hat_face:

**Pull request merged**